src/ApiConnectors/BankTransactionApiConnector.php

```diff
@@ -21,21 +21,21 @@ class BankTransactionApiConnector extends BaseApiConnector
      */
     public function send(BankTransaction $bankTransaction): BankTransaction
     {
-        $responses = $this->sendAll([$bankTransaction]);
+        $result = $this->sendAll([$bankTransaction]);
```

Comment:
I'd give this a bit more of a useful variable name. `$bank_transaction_responses`. `foreach ($bank_transaction_responses as $bank_transaction_response)`.